internal/store/device_test.go

```diff
@@ -266,5 +257,38 @@ var _ = Describe("DeviceStore create", func() {
 			Expect(dev.Status.Conditions).ToNot(BeNil())
 			Expect((*dev.Status.Conditions)[0].Type).To(Equal(api.EnrollmentRequestApproved))
 		})
+
+		It("UpdateDeviceOwner", func() {
+			called := false
+			callback = DeviceStoreCallback(func(before *model.Device, after *model.Device) {
+				called = true
+			})
+			err := store.Device().UpdateOwner(ctx, orgId, "mydevice-1", "newowner", callback)
+			Expect(err).ToNot(HaveOccurred())
+			Expect(called).To(BeTrue())
+			dev, err := store.Device().Get(ctx, orgId, "mydevice-1")
+			Expect(err).ToNot(HaveOccurred())
+			Expect(*dev.Metadata.Owner).To(Equal("newowner"))
+		})
+
+		It("UpdateDeviceAnnotations", func() {
+			firstAnnotations := map[string]string{"key1": "val1", "key2": "val2"}
+			err := store.Device().UpdateAnnotations(ctx, orgId, "mydevice-1", firstAnnotations, nil)
+			Expect(err).ToNot(HaveOccurred())
+			dev, err := store.Device().Get(ctx, orgId, "mydevice-1")
+			Expect(err).ToNot(HaveOccurred())
+			Expect(dev.Metadata.Annotations).ToNot(BeNil())
+			Expect(len(*dev.Metadata.Annotations)).To(Equal(2))
```

Comment:
Some of the coolnes of ginkgo would let you do this too:
```
    Expect(*dev.Metadata.Annotations).To(HaveLen(2))
```

Reply:
and same in other places where we use len

Reply:
Nice!